java/pom.xml

```diff
@@ -39,7 +39,7 @@
     <dep.fbs.version>1.9.0</dep.fbs.version>
     <dep.flatc.version>1.9.0</dep.flatc.version>
     <arrow.vector.classifier></arrow.vector.classifier>
-    <forkCount>2</forkCount>
+    <forkCount>0</forkCount>
```

Comment:
Thanks for fixing this problem. 

IMO, reducing the value of forkCount seems to be the correct direction of solving the problem. 
However, it also reduces the parallelism of running tests. 
The original value (2) is already small enough. I am not sure if we should further reduce it.

Reply:
The bug still hasn't been fixed -- reducing the forkCount to 0 simply is allowing us to see the segfault / core dump that's happening...